docs/API.md

```diff
@@ -0,0 +1,309 @@
+**Show All Supported Coins(Assets)**
+
+\----
+
+Returns all supported coins(Assets)
+
+\* **URL**
+
+ /coins
+
+\* **Method:**
+
+   `GET`
+
+\*  **URL Params**
+
+   **Optional:** 
+
+​     `id=[string]`
+
+\* **Success Response:**  
+
+  \* **Code:** 200 <br />
+
+​    **Content:**
+```json
+[
+    {
+        "id": "988f08b3b21e372",
+        "blockchain": {
+            "platform": "ethereum",
+            "network": "ropsten"
+        },
+        "symbol": "eth",
+        "decimals": 18,
+        "meta": {}
+    },
+    {
+        "id": "b21e3988f08b372",
+        "blockchain": {
+            "platform": "bitcoin",
+            "network": "mainnet"
+        },
+        "symbol": "btc",
+        "decimals": 8,
+        "meta": {}
+    }
+]
+```
+
+\* **Sample Call:**
+```
+curl --request GET 'localhost:8080/coins'
+```
+
+---
+
+**Show A Coin(Asset)**
+
+\----
+
+Returns data about a coin
+
+\* **URL**
+
+ /coin/{id}
+
+\* **Method:**
+
+   `GET`
+
+\* **Success Response:**  
+
+  \* **Code:** 200 <br />
+
+​    **Content:**
+```json
+{
+    "id": "988f08b3b21e372",
+    "blockchain": {
+        "platform": "ethereum",
+        "network": "ropsten"
+    },
+    "symbol": "eth",
+    "decimals": 18,
+    "meta": {}
+}
+```
+
+\* **Sample Call:**
+```
+curl --request GET 'localhost:8080/coins/988f08b3b21e372'
+```
+
+---
+
+**Show Wallets**
+
+\----
+
+Returns data about all wallets
```

Comment:
At first, I thought it returns data about **all wallets**.
But, it seems like returning data about **a wallet** for all coin.

`Returns data about all wallets` confused me. 

So, I suggest `Show wallet` or `Show wallet data` to title.
And, `Returns data about wallet` to explanation.